testapp/build.gradle

```
@@ -48,6 +48,7 @@ android {
     def debugManifestPlaceholders = [
             baseUrl                 : property("MINIAPP_SERVER_BASE_URL") ?: "https://www.example.com/",
             isPreviewMode           : property("IS_PREVIEW_MODE") ?: true,
+            requireSignatureVerification : property("REQUIRE_SIGNATURE_VERIFICATION") ?: true,
```

default should be `false` there
```suggestion
            requireSignatureVerification : property("REQUIRE_SIGNATURE_VERIFICATION") ?: false,
```